local-volume/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,49 @@
+common:
+#
+# Defines the namespace where provisioner runs
+#
+  namespace: default
+configmap:
+#
+# Defines the name of configmap used by Provisioner
+#
+  configMapName: "local-provisioner-config"
+#
+# Defines storage classes in a format:
+#
+#  storageClass:
+#    - {storage class name}:
+#        hostDir: "{path on the host where local volume of this storage class are mounted}"
+#        mountDir: "{Optional path where a local volume will be mounted inside of a container}"
+#
+# Note: At least one storage class must be configured
+  storageClass:
+    - fast-disks:
+        hostDir: /mnt/fast-disks
+        mountDir: /local-fast-disks
+#
+daemonset:
+#
+# Defines the name of a Provisioner
+#
+  name: "local-volume-provisioner"
+#
+# Defines Provisioner's image name including container registry.
+#
+  image: quay.io/external_storage/local-volume-provisioner:latest
+#
+# Defines Image download policy, see kubernet3es documentation for available values.
+#
+  imagePullPolicy: Always
+#
+# Defines a name of the service account which Provisioner will use to communicate with API server.
+#
+  serviceAccount: local-storage-admin
+#
+# If configured KubeConfigEnv will (optionally) specify the location of kubeconfig file on the node.
+#  kubeConfigEnv: KUBECONFIG
+#
+# List of node labels to be copied to the PVs created by the provisioner in a format:
+#
+#  nodeLabels:
+#     label1: label1-value
```

Comment:
@dhirajh - can you please recommend a useful label here to make the example a little more realistic?

Reply:
done, I took it from @dhirajh's PR